boards/arm/atsame54_xpro/doc/index.rst

```diff
@@ -0,0 +1,176 @@
+.. _atsame54_xpro:
+
+SAM E54 Xplained Pro Evaluation Kit
+###################################
+
+Overview
+********
+
+The SAM E54 Xplained Pro evaluation kit is ideal for evaluation and
+prototyping with the SAM E54 Cortex®-M4F processor-based
+microcontrollers. The kit includes Atmel’s Embedded Debugger (EDBG),
+which provides a full debug interface without the need for additional
+hardware.
+
+.. image:: img/atsame54_xpro.png
+     :width: 500px
+     :align: center
+     :alt: ATSAME54-XPRO
+
+Hardware
+********
+
+- ATSAME54P20A ARM Cortex-M4F processor at 120 MHz
+- 32.768 kHz crystal oscillator
+- 12 MHz crystal oscillator
+- 1024 KiB flash memory and 256 KiB of RAM
+- One yellow user LED
+- One mechanical user push button
+- One reset button
+- On-board USB based EDBG unit with serial console
+- One QTouch® PTC button
+- 32 MiB QSPI Flash
+- ATECC508 CryptoAuthentication™  device
+- AT24MAC402 serial EEPROM with EUI-48™ MAC address
+- Ethernet
+	- RJ45 connector with built-in magnetics
```

Comment:
add a blank line before the first item in a (sub) list, and don't indent more than 3 spaces from the parent list item.